lib/packages/docutils/rst.nim

```diff
@@ -1151,6 +1151,17 @@ proc toOtherRole(n: PRstNode, kind: RstNodeKind, roleName: string): PRstNode =
 proc parsePostfix(p: var RstParser, n: PRstNode): PRstNode =
   var newKind = n.kind
   var newSons = n.sons
+
+  proc finalizeInterpreted(node: PRstNode, newKind: RstNodeKind,
+                           roleName: string): PRstNode =
```

Comment:
Use `.nimcall` to prevent the closure overhead.

Reply:
thanks, changed